packages/driver/src/cypress/log.ts

```diff
@@ -505,8 +505,11 @@ class Log {
 
       consoleObj[key] = _this.get('name')
 
+      // in the case a log is being recreated from the specbridge to the primary, consoleProps may be an Object
```

Comment:
```suggestion
      // in the case a log is being recreated from the cross-origin spec bridge to the primary, consoleProps may be an Object
```

Just thinking about if I someone unfamiliar with cross-origin stuff, having the 'cross-origin' keyword might help with seeking more information on what this means.

Reply:
updated in [02cb280](https://github.com/cypress-io/cypress/pull/20949/commits/02cb2805e898cf68d5c351cec8afc10483083afd)